src/main/java/org/folio/circulation/domain/User.java

```diff
@@ -35,17 +35,17 @@ public boolean canDetermineStatus() {
     return !representation.containsKey("active");
   }
 
-  public Boolean isInactive() {
+  public boolean isInactive() {
     return !isActive();
   }
 
-  Boolean isActive() {
-    final Boolean active = representation.getBoolean("active");
+  boolean isActive() {
+    final boolean active = representation.getBoolean("active");
```

Comment:
The same as for previous, is it required property, if no then we need a default value for it? 
```suggestion
    final boolean active = representation.getBoolean("active", false);
```